src/configLoaders/DefaultConfigLoader.ts

```diff
@@ -0,0 +1,20 @@
+import ConfigLoader from './ConfigLoader';
+import * as path from 'path';
+
+export default class DefaultConfigEditor implements ConfigLoader {
+  private fs: any;
+  private projectRoot: string;
+
+  constructor(projectRoot: string, fs: any) {
```

Comment:
Could you set the properties in the constructor using ` constructor(private projectRoot: string, private fs: any) {`

Reply:
I really prefer it the way it is right now, it makes the code much clearer to people that have less experience with TypeScript. I mean, what other programming language assign their private variables in the constructor?